src/components/atoms/NavLink.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { Children } from 'react'
+import { useRouter } from 'next/router'
+import cx from 'classnames'
+import Link, { LinkProps } from 'next/link'
+
+type NavLinkProps = React.PropsWithChildren<LinkProps> & {
+  activeClassName?: string
```

Comment:
proc to tu vubec je? kdyz to pro aktivni link stejne nepouzivas a mas to copy paste - return div className="text-blue-900 ...."

Reply:
Tady stacilo vzit ten router as path, to jsi udelal dobre. A pak jen renderovat vzdycky Link, kteremu bud das Active classNames nebo nonactive classNames